karton/dashboard/templates/index.html

```diff
@@ -19,7 +19,14 @@ <h3 class="text-center">binds</h3>
         <td>
           <a href="queue/{{ queue_name }}">{{ queue_name }}</a>
           <div>
-            <span class="badge badge-info">{{ queue.bind.version }}</span>
+            <span class="badge badge-info">
+              📦{{ queue.bind.version }}
+            </span>
```

Comment:
Maybe add onmouseover tooltips to both of them? To make it more obvious what these versions are and how to interpret this.

Just "karton-core library version" and "version of installed package for this service" (or similar) should be enough I think.

Reply:
![image](https://user-images.githubusercontent.com/2192416/104187762-91620300-5418-11eb-9b06-45eac169b53f.png)

`title` seems to do the job